### PR TITLE
BC-1617 - ukrainian feature flag

### PR DESCRIPTION
**Pull request by odalys-dataport**
# Short Description

## Links to Ticket and related Pull-Requests
hpi-schul-cloud/schulcloud-client#2816
hpi-schul-cloud/schulcloud-server#3400
hpi-schul-cloud/dof_app_deploy#218

## Checklist before merging

- [ ] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [ ] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/components/legacy/PopupIconInitials.vue [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
-	>
-		<div class="icon">{{ initials }}</div>
+	<div class="popup">
+		<base-button
+			v-outside-click="removePopup"
+			type="button"
+			class="icon-button button is-medium is-text is-icon"
+			data-testid="initials"
+			role="menu"
+			:aria-label="`Aktuelle/r Benutzer:in ${firstName} ${lastName} ${userRole}`"
```
Comment (SevenWaysDP) [inline src/components/legacy/PopupIconInitials.vue:9]: "Aktuelle/r Benutzer:in" move to locales

File: src/components/molecules/LanguageMenu.vue [hunk 2]
```diff
@@ -1,9 +1,11 @@
 <template>
-	<v-list>
+	<v-list role="menuitem">
 		<v-list-item
 			dense
+			role="menu"
 			:data-testid="`selected-language-${selectedItem.language}`"
-			@click.stop="toggleMenu"
+			:aria-label="`Ausgewählte Sprache ${selectedItem.language}`"
```
Comment (SevenWaysDP) [inline src/components/molecules/LanguageMenu.vue:7]: "Aktuelle/r Benutzer:in" move to locales

"Ausgewählte Sprache"
"Selected Language"
"Idioma seleccionado",
"Вибрана мова"

**Review by SevenWaysDP — CHANGES_REQUESTED** [review]

File: src/components/legacy/PopupIconInitials.vue (hunk 1 above)
Comment (uidp) [reply src/components/legacy/PopupIconInitials.vue]: Ok, fixed.

File: src/components/molecules/LanguageMenu.vue (hunk 2 above)
Comment (uidp) [reply src/components/molecules/LanguageMenu.vue]: Ok, fixed.

**Review by SevenWaysDP — APPROVED** [review]

**Pull request merged**